napari/_qt/qt_main_window.py

```diff
@@ -1611,13 +1611,9 @@ def _teardown(self):
 
     def close(self):
         """Close the viewer window and cleanup sub-widgets."""
-        # Someone is closing us twice? Only try to delete self._qt_window
-        # if we still have one.
-        if hasattr(self, '_qt_window'):
-            self._teardown()
-            self._qt_viewer.close()
-            self._qt_window.close()
-            del self._qt_window
```

Comment:
worth putting this `del` back?

Reply:
no. This `del` causes segfaults.

Reply:
oh. crap is worst than expected...

Reply:
maybe push these changes to the 6.5.1 fix branch and see if >6.5 helps?

Reply:
again lambda in face...